assets/src/blocks/Covers/CampaignCovers.js

```diff
@@ -0,0 +1,45 @@
+const { __ } = wp.i18n;
+
+export const CampaignCovers = ({ covers, covers_view, row, loadMoreCovers }) => {
+  const rowAmount = 3;
```

Comment:
This could be read as amount per row, or amount of rows. If we call it`amountPerRow` it won't have that ambiguity.